atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/sweep/queue/MultiTableSweepQueueWriter.java

```diff
@@ -41,30 +39,13 @@ default void enqueue(Map<TableReference, ? extends Map<Cell, byte[]>> writes, lo
      */
     void enqueue(List<WriteInfo> writes);
 
-    /**
-     * This method must be implemented if asynchronous initialization is necessary for the implementation. This is
-     * generally the case if the transaction manager allows asynchronous initialization since there is no guarantee
-     * the underlying kvs is ready at object creation time.
-     *
-     * @param txManager the transaction manager performing the callback
-     */
-    @Override
-    default void initialize(TransactionManager txManager) {
```

Comment:
checked in large internal product that asynchronous initialization is NOT necessary for the implementation there (and in our usage, the way in which TM initialisation works means that it is also not necessary for us)